windows/unicode.mpl

```diff
@@ -9,39 +9,40 @@
 "String.makeStringView" use
 "control.Nat16"         use
 "control.assert"        use
+"control.keep"          use
+"control.swap"          use
 
 "kernel32.CP_UTF8"              use
 "kernel32.MB_ERR_INVALID_CHARS" use
 "kernel32.MultiByteToWideChar"  use
 
 utf16: [
   source: makeStringView;
-  destSize:
+  targetNat16Count:
     0
     0nx
     source.size
     source.data storageAddress
     MB_ERR_INVALID_CHARS
-    CP_UTF8 MultiByteToWideChar
+    CP_UTF8
+    MultiByteToWideChar
   ;
+  [targetNat16Count 0 = ~] "MultiByteToWideChar failed" assert
 
-  [destSize 0 = ~] "MultiByteToWideChar failed" assert
+  result: Nat16 Array [targetNat16Count 1 + swap.enlarge] keep;
```

Comment:
I suggest adding a space after `swap`, so that it does not look like a method call.

Reply:
Done - https://github.com/Matway/mpl-sl/pull/74/commits/e16aa2149ee5c4a266bb9a9a6907d5216e28925a.